### ISSUE TITLE
Integ tests failing for 1.3.0

### ISSUE THREAD

**Issue by ohltyler** [opensearch-project/asynchronous-search#90]
When running the integ tests for this plugin in 1.3.0, seeing failures related to JUnit versioning. Not seeing issues for any other plugins (Jenkins job `ohltyler-integ-test-stable #73`).

Trace:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
   > Conflict(s) found for the following module(s):
       - junit:junit between versions 4.13.2 and 4.12
     Run with:
         --scan or
         :dependencyInsight --configuration testCompileClasspath --dependency junit:junit
     to get more insight on how to solve the conflict.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 21s
```

**Comment by ohltyler** [comment]
@eirsep Hi Surya, can I get an ETA on when this can be fixed? Thank you!

**Comment by ohltyler** [comment]
Fixed in #91
